### PR TITLE
Add time taken to address request

### LINKED ISSUE #5: Display time taken to address request
Let the users know the time taken to address their request for logistical purposes.

### PR DESCRIPTION
**Pull request by pedroandrade**
Hi @t0xic0der adding a PR related to issue #5 

this one calculates the request time until the function was completely processed

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -1,8 +1,9 @@
 package main
 
 import (
-	"./requestor"
 	"flag"
+
+	"github.com/t0xic0der/prudence/requestor"
```
Comment (gridhead) [inline main.go]: You would want to restore it back to the way it was before as `requestor` is packaged within `prudence` and there is no need to fetch it like this.

File: requestor/worldstate.go [hunk 2]
```diff
@@ -1,14 +1,19 @@
...
 
 import (
-	"../colorformat"
 	"fmt"
-	"github.com/tidwall/gjson"
 	"io/ioutil"
 	"net/http"
+	"time"
+
+	"github.com/t0xic0der/prudence/colorformat"
```
Comment (gridhead) [inline requestor/worldstate.go]: Here too. Import `colorformat` like it was done before. Do let me know if you face issues while doing so.

File: requestor/worldstate.go [hunk 3]
```diff
@@ -141,3 +156,7 @@ func DailyDeals(platform string) {
 		fmt.Println("The information could not be fetched")
 	}
 }
+
+func timeTracking(startTime time.Time) {
+	fmt.Println("\n" + colorformat.ForeGrnRglr("Request Time") + " ... " + time.Since(startTime).Round(time.Millisecond).String() + "\n")
```
Comment (gridhead) [inline requestor/worldstate.go]: Any reason as to why the time is fetched in milliseconds? :sweat_smile:

**Review by gridhead — CHANGES_REQUESTED** [review]
Thanks for the work done so far @pedroandrade. Really appreciate it.

You are requested to make the changes suggested before I head in to review again.

File: requestor/worldstate.go (hunk 3 above)
Comment (pedroandrade) [reply requestor/worldstate.go]: it's just for formatting time in milliseconds, there is an example of how it works -> https://play.golang.org/p/tscRrEizKY8

Comment (gridhead) [reply requestor/worldstate.go]: All right. That makes sense.

**Review by gridhead — CHANGES_REQUESTED** [review]
A minor change that I would request for you to introduce here. There's an inconsistency in newline creation while displaying the request times. When it comes to `--conclave` and `--drvodeal`, two newlines are created before the time is displayed while for other options, only one newline is made. 

Please make them consistent to only one newline.

**Review by gridhead — CHANGES_REQUESTED** [review]
I will request for a rebase to resolve the merge conflicts.

**Comment by pedroandrade** [comment]
hey @t0xic0der no problem  :D, I did it, I saw that you're using spectrum instead of colorformat package, it's very cool

**Comment by pedroandrade** [comment]
@t0xic0der any other comment here?

**Comment by gridhead** [comment]
> hey @t0xic0der no problem :D, I did it, I saw that you're using spectrum instead of colorformat package, it's very cool

Thanks. I could use some reduced redundancy so I decoupled the inbuilt coloring package for the one I wrote.

> @t0xic0der any other comment here?

So sorry. The mail notification just got lost in the volumes of the ones I receive. I would get to reviewing this shortly.

**Review by gridhead — APPROVED** [review]
Looks good to me. Approving.

**Pull request merged**